content/member-handbook.md

```diff
@@ -147,6 +147,186 @@ That said, here are some things we might do, in rough order of severity:
+information may be asked on a case-by-case basis to either include more
+information in their postings or to stop posting.
+
+  - Employer Name:  As stated above, the employer name is not required, however
+    it is highly encouraged that all job postings SHOULD include the name of the
+    employer where the candidate would be working.
+  - Salary: Job postings SHOULD include a reasonable salary range.  It's better
+    to guess at this than to omit it, unless you really have no idea at all.
+  - Contract Margins:  For contract and contract-to-hire positions, recruiters
+    SHOULD include their hourly or fixed cost margins that are added to the
+    candidates salary.
+  - Benefits: Job postings SHOULD include a list of available benefits, such as
+    amount of PTO, health insurance, paternity or maternity leave, educational
+    benefits, etc.
+  - Number of positions: A job posting SHOULD disclose how many candidates are
+    expected to be hired for the given position.
+  - Type of work: Many job postings include a list of technology skills and
+    requirements, but do not speak to the nature of the work.  While not
+    required, job postings SHOULD include a description of the type of work to
+    be performed.
+
+##### Additional Considerations
+
+Job postings on the STL Tech Slack should be crafted to speak to our community.
+There is no prohibition against merely copying information from an existing job
+posting, but we encourage members of the STL Tech community go personalize the
+message for the audience.  In particular:
+
+  - Leverage tools like [joblint](http://joblint.org) to ensure your job posting
+    does not contain language that would exclude marginalized people.
+  - Avoid terms like "rockstar" and "ninja".
+  - Understand [what your job posting is
+    saying](https://where.coraline.codes/blog/not_applicable/) and try to be
+    considerate of using inclusive language.
+  - Participate in the community, don't just stop by to dump a job listing and
+    then go silent.
+
+
+### Aggregate Job Postings
+
+Recruiters, and companies who are looking to quickly grow new departments or to
+significantly expand, may wish to skip posting individual listings, and instead
+state that they have several different roles available across a broad range of
+technologies.  For the most part, postings like this should be treated similarly
+to For Hire postings.  They should be short, inclusive of the types of work
+available, and limited to being posted weekly.  For external recruiters,
+aggregate job postings must also disclose that the poster is an external
+recruiter, as well as the type(s) of roles that are available (contract, direct
+hire, contract to hire).  Additionally, for anyone posting aggregate listings,
+we ask that the job poster be available via the STL Tech Slack to answer any
+questions.  Anyone creating  aggregate postings then responding to questions
+with a request for a phonecall / email will be banned.
```

Comment:
This is a little harsh. This is not an one-and-done bannable thing. People tend to learn what's acceptable over time. If someone does it once, warn them. If they do it again, a ban is fine. But this is a longish document with a few cliffs to fall off already.

Reply:
I'd suggest tweaking to "may be banned" or "may be punished, up to and including banning" or something like that.